### PR TITLE
Add license info to `setup.py` so it will show up in wheel installs

### PR DESCRIPTION
**Pull request by itsbenweeks**
Just unable to see the license info from packaging. This will correct that.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ def get_version(module='pylsp'):
     name='python-lsp-server',
     version=get_version(),
     description='Python Language Server for the Language Server Protocol',
+    license="MIT",
+    license_file="LICENSE"
```
Comment (ccordoba12) [inline setup.py]: A comma is missing here at end.

**Review by ccordoba12 — COMMENTED** [review]
Hey @itsbenweeks, thanks a lot for your contribution! I left a small comment for you, otherwise looks good to me.

**Comment by ccordoba12** [comment]
@itsbenweeks, do you need help to finish this one? We can give you hand so we can have this change in our next version.

File: setup.py (hunk 1 above)
Comment (itsbenweeks) [reply setup.py]: Oh, sorry! Will fix that ASAP.

**Comment by itsbenweeks** [comment]
Sorry about the slow response. Got this committed and up to date, now.

**Comment by ccordoba12** [comment]
Thanks @itsbenweeks! But since we moved our metadata from `setup.py` to `setup.cfg`, you need to do the same here.

Just take a look at PR #84 to see how to do that.

**Comment by itsbenweeks** [comment]
Thanks for letting me know, @ccordoba12! It's been changed accordingly.

**Review by ccordoba12 — APPROVED** [review]
Thanks for your help with this @itsbenweeks!

**Pull request merged**